iphone/Classes/MediaModule.m

```diff
@@ -1105,8 +1106,20 @@ - (void)openPhotoGallery:(id)args
 {
   ENSURE_SINGLE_ARG_OR_NIL(args, NSDictionary);
   ENSURE_UI_THREAD(openPhotoGallery, args);
-  [self showPicker:args isCamera:NO];
+
+  NSArray *types = (NSArray *)[args objectForKey:@"mediaTypes"];
+#if IS_SDK_IOS_14
+  if ([TiUtils isIOSVersionOrGreater:@"14.0"] && ([args objectForKey:@"selectionLimit"] || [TiUtils boolValue:[args objectForKey:@"allowMultiple"] def:NO])) {
```

Comment:
Can we maybe always use it on iOS 14? The old image picker is now deprecated and should be replaced for iOS 14+.

Reply:
In new PHPickerViewController, there is no option for image editing(may be included in future releases). I think this is important feature and we can not remove it in minor release.